src/tests/Microsoft.Agents.BotBuilder.Tests/TurnContextTests.cs

```diff
@@ -529,6 +530,96 @@ public async Task SendActivityAsync_ShouldThrowExceptionInOnSend()
             await Assert.ThrowsAsync<Exception>(() => context.SendActivityAsync(TestMessage.Message()));
         }
 
+        [Fact]
+        public async Task SendActivityAsync_ShouldCreateActivityWithParameters()
+        {
+            var adapter = new SimpleAdapter();
+            var context = new TurnContext(adapter, new Activity());
+            Assert.False(context.Responded);
+            var textMessage = "test activity created with parameters";
+            var response = await context.SendActivityAsync(textMessage, textMessage, InputHints.AcceptingInput);
+
+            Assert.True(context.Responded);
+        }
+
+        [Fact]
+        public async Task SendActivitiesAsync_ShouldThrowExceptionOnEmptyActivitiesList()
+        {
+            var adapter = new SimpleAdapter();
+            var context = new TurnContext(adapter, new Activity());
+            var exceptionMessage = "Expecting one or more activities, but the array was empty.";
+           
+            var exception = await Assert.ThrowsAsync<ArgumentException>(async () => await context.SendActivitiesAsync(Array.Empty<Activity>()));
```

Comment:
This can be simplified.
```suggestion
            var exception = await Assert.ThrowsAsync<ArgumentException>(async () => await context.SendActivitiesAsync([]));
```

Reply:
done, thanks